Index: my_dream.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Список моїх мрій:\n1.\n2.\n3.\n4.\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/my_dream.txt b/my_dream.txt
--- a/my_dream.txt	(revision 5cdbc13f91b45c8dc5dc2af526f02ff498271bca)
+++ b/my_dream.txt	(date 1784636583563)
@@ -1,5 +1,1 @@
-Список моїх мрій:
-1.
-2.
-3.
-4.
+Моя перша мрія
\ No newline at end of file
